.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedDateTimeParser.cs

```diff
@@ -514,14 +521,15 @@ public List<DateTimeParseResult> DateTimeResolutionForSplit(DateTimeParseResult
 
             foreach (var p in res)
             {
-                if (p.Value is Dictionary<string, string>)
+                if (p.Value is Dictionary<string, string> dictionary)
                 {
-                    var value = new Dictionary<string, string>();
+                    var value = new Dictionary<string, object>();
```

Comment:
Using object is probably not the best design. Can we use a concrete class in here?

Reply:
and elsewhere...

Reply:
I agree that the `object` isn't the optimal solution here, but since changing this will lead to a great effort to a big refactor, I suggest to do this as a further improvement.

Reply:
Add this to the same work item as that signature refactor, ok?